packages/twenty-front/src/pages/settings/developers/webhooks/components/SettingsDevelopersWebhookDetail.tsx

```diff
@@ -148,43 +176,50 @@ export const SettingsDevelopersWebhooksDetail = () => {
         <Section>
           <H2Title
             title="Filters"
-            description="Select the event you wish to send to this endpoint"
+            description="Select the events you wish to send to this endpoint"
+          />
+          {operations.map((operation, index) => (
+            <StyledFilterRow key={index}>
+              <Select
+                fullWidth
+                dropdownId={`object-webhook-type-select-${index}`}
+                value={operation.object}
+                onChange={(object) => updateOperation(index, 'object', object)}
+                options={fieldTypeOptions}
+              />
+              <Select
+                fullWidth
+                dropdownId={`operation-webhook-type-select-${index}`}
+                value={operation.action}
+                onChange={(action) => updateOperation(index, 'action', action)}
+                options={[
+                  { value: '*', label: 'All Actions' },
+                  { value: 'create', label: 'Create' },
+                  { value: 'update', label: 'Update' },
+                  { value: 'delete', label: 'Delete' },
+                ]}
+              />
+
+              <IconButton
+                onClick={() => removeOperation(index)}
+                variant="tertiary"
+                size="medium"
+                Icon={IconTrash}
+              />
+            </StyledFilterRow>
+          ))}
+          <StyledAddOperationButton
+            variant="tertiary"
+            title="Add Operation"
+            Icon={IconPlus}
+            onClick={addOperation}
```

Comment:
This does not match the figma design. You have 2 choices there:
- Remove that button, and:
  - Automatically add an new operation when the first Operation is updated
  - The first operation should have an emptyOption parameter with `{ value: '*', label: 'All Objects' }` for object `<Select` and `{ value: '*', label: 'All Actions' }` for action `<Select`
  - This new operation should have an emptyOption parameter with `{value: null, label: 'Object'}` for object `<Select` and `{value: null, label: 'Action'}` for action `<Select` (they will not be selectionable as they are not in the options array)
  - Before saving, remove those null values from `operations`
  - Automatically add an new operation when the last `operation` is updated, and so on
- suggest your design to @Bonapara and validate it with him

Reply:
Ah, okay understood!